src/atoms/Text/Text.native.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+import theme from '../../tokens/theme';
+import { Text as NativeText } from 'react-native';
+import { getAllColorNames, resolveColorFromString } from '../../utils/colors';
+import automation from '../../_helpers/automation-attributes';
+
+const fontFamilies = {
+  light: `Lato-Regular`,
```

Comment:
Shouldn't it be `light: Lato-Light`?

Reply:
DS only has Regular & Bold

Reply:
why are we keeping it here? any reasons?

Reply:
Because it will not be used anywhere else but here